core_protocol_working_group/meetings/yyyy-mm-dd.md

```diff
@@ -0,0 +1,28 @@
+# Meeting 1
```

Comment:
is there a reason to repeat the template here ?

Reply:
💡 thanks for the comment. I overlooked the template file [`meeting_template.md`](https://github.com/onflow/Flow-Working-Groups/blob/main/meeting_template.md) in the root folder. 

Removed my template file [`core_protocol_working_group/meetings/yyyy-mm-dd.md`](https://github.com/onflow/Flow-Working-Groups/pull/9/files/1210294cd2458bc8b4d5f6f99aaa019a6bd502ca#diff-ed896756e2b1a0ce9d61c53bdbf9fb6612117c48e6ef088499732612e6e4f68f) in https://github.com/onflow/Flow-Working-Groups/pull/9/commits/821ca7952453cd5fa49ce5109921c489caf35cc2